docs/design/design-reviews/recipes/sprint-spike-template.md

```diff
@@ -0,0 +1,33 @@
+# Spike: {Name}
+
+- **Conducted by:** {Names and at least one email address for follow-up questions}
+- **Backlog Work Item:** {Link to the work item to provide more context}
+- **Sprint**: {Which sprint did the study take place. Include sprint start date}
+
+## Goal
+
+Describe what question(s) the spike intends to answer and why.
+
+## Method
+
+Describe how the team will uncover the answer to the question(s) the spike intends to answer. For example:
+
+1. Build prototype to test.
```

Comment:
nit: GitHub flavored Markdown (and other renderers) will render lists of repeated `1.`s as ascending. If a list becomes very long, this can speed up list insertion in future edits.

Reply:
fixed! updated the trade study too to use `1.` for ordered lists.